spec/integration/manifest.spec.yml

```diff
@@ -90,6 +90,7 @@ spec:
           - web_application_firewall
           - websphere
           - workplace_search
+          - cloudsecurity_cdr
```

Comment:
@nimarezainia are you ok with adding this category?

Reply:
Not sure how these buttons are configured. This is probably coded in Kibana/Fleet.